### ISSUE TITLE
SmartCity-Accessibility Server Side #431 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#275]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [yaelAmitay] 2a09aa2bf06f4a873bbf5a481442f0807a08eae9 - add guiava dependancy #253

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/431/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
